src/app/services/copy/copy.service.ts

```diff
@@ -0,0 +1,23 @@
+import { Injectable } from '@angular/core'
+
+@Injectable({
+  providedIn: 'root'
+})
+export class CopyService {
+  public copyToClipboard(val: string) {
+    // This would be a nicer solution, but it doesn't work in Safari
+    // (navigator as any).clipboard.writeText(val)
```

Comment:
Is safari support necessary?

Also, I think in the beacon-sdk we use this or something similar and it also works in Safari.

Reply:
we do not need to support safari for now